tests/src/test/scala/cats/tests/XorTests.scala

```diff
@@ -46,6 +47,12 @@ class XorTests extends CatsSuite {
     checkAll("Eq[ListWrapper[String] Xor ListWrapper[Int]]", SerializableTests.serializable(Eq[ListWrapper[String] Xor ListWrapper[Int]]))
   }
 
+  {
+    implicit val L = ListWrapper.semigroup[String]
```

Comment:
This line is creating a local (to the surrounding `{ ... }` block) `Semigroup[ListWrapper[String]]` instance. We are taking this local instance approach in some tests, because we want to make sure that things work given the minimum required instances. For example, does this work when we have a `Semigroup[ListWrapper[String]]` but no `Monoid[ListWrapper[String]]`?

In this case, the `SemigroupK` instance you created doesn't have any requirements (specifically, no `Semigroup[L]` instance is required). So we shouldn't have to take this approach here. We should be able to just add the 2 `checkAll` tests around line 34 without creating a separate block to house locally-scoped instances.

I'd imagine all of these locally-scoped instances are somewhat terrifying for someone who is used to Haskell :)